usage/rest-server/src/test/java/org/apache/brooklyn/rest/BrooklynRestApiLauncher.java

```diff
@@ -314,6 +314,7 @@ public static void main(String[] args) throws Exception {
     public static Server startRestResourcesViaFilter() {
         return new BrooklynRestApiLauncher()
                 .mode(StartMode.FILTER)
+                .forceUseOfDefaultCatalogWithJavaClassPath(true)
```

Comment:
Well this is the only way I know to see something in the catalog and test brooklyn.  `$ bin\brooklyn launch` uses `.forceUseOfDefaultCatalogWithJavaClassPath(true)`. Don't you want the test to be identical with the way is used? I think it slows down the build (the tests), but has no other consequence.

Do you have any suggestions?